tests/unit/parallel/algorithms/copy.cpp

```diff
@@ -387,12 +387,11 @@ void copy_bad_alloc_test()
 
 int hpx_main(boost::program_options::variables_map& vm)
 {
-    unsigned int seed = (unsigned int)std::time(nullptr);
+    unsigned int seed = std::random_device{}();
     if (vm.count("seed"))
         seed = vm["seed"].as<unsigned int>();
 
     std::cout << "using seed: " << seed << std::endl;
-    std::srand(seed);
```

Comment:
If `std::srand` is removed `seed` is completely unused, no? Either `std::srand` should stay or `seed` (or a generator initialized with `seed`) should be passed one way or another down to the tests (which directly use `std::rand` now). What am I missing?

Reply:
@msimberg I have not completed this file(see this still uses `std::rand`).I have only covered [these](https://paste.ubuntu.com/p/4ZRfBy997t/) 30 files.As soon as this PR merges I will file another PR for next 30 files(including this).

Also, this file commit has happened accidentally because I have started my work for another 30 files but seeing some issues regarding `std::random_device` I stopped and I have done hard reset to come back to the previous commit but my mistake this change still happened here.But I will soon fix this file in another PR as soon as this merges(I assure within a day or two).If this is not acceptable then I will fix it right away :)

Reply:
@Anushi1998 Okay, no problem in that case! Just make sure this change doesn't get included in this PR. Thanks for clarifying.

Reply:
Sure @msimberg

Reply:
@msimberg I have reverted the changes.Thanks for figuring out :)